blog: merge time vs review complexity

GregSutcliffe opened the pull request:
@tbrisker hopefully this helps your Discourse discussion on review complexity :)

GregSutcliffe committed:
blog: merge time vs review complexity

evgeni commented on _posts/2019-03-04-merge-time-review-complexity.md:
```diff
@@ -0,0 +1,228 @@
+In this post I'm going to dig into some of the data we have on GitHub about how
+our community functions. In particular, I'm looking to investigate some of the
+issues raised
+[here](https://community.theforeman.org/t/improving-pr-merge-velocity/12992) to
```
stop. linking. to. "here". :)

`investigate some of the [issues raised on Discourse](…)`

GregSutcliffe replied:
Sorry, bad habit. Fixed.

GregSutcliffe commented:
Thanks to @tbrisker for some discussion in IRC. Updated with a reworded section on how "typical values" affect *only* the drawing of the graphs (the model uses the whole range) and changed the typical values to get a better feel for the effects in play. Much happier with the final graph now!

tbrisker reviewed (approved):
Thanks for the great work @GregSutcliffe !
